### ISSUE TITLE
Trailing forward slash in entry point addressing the API's root

### ISSUE THREAD

**Issue by SebastianKotstein** [restful-ma/rest-ruler#52]
**Describe the bug**
RESTRuler detects violations of:
- A trailing forward slash (/) should not be included in URIs
- Forward slash separator (/) must be used to indicate a hierarchical relationship

... in the following OpenAPI snippet:
```
paths:
  /:
    get:
      tags: 
      - Entry Point
      summary: Endpoint serving as entry point for this API
```
As described in `summary`, `GET /` is an entry point for an API and represents a _docroot_, i.e., _the hierarchical ancestor of all other resources within a REST API's model_ (Massé, 2011). To _Minimize the number of advertised "entry point" API URIs_ (to one), Massé suggests to expose this docroot and make it accessible via the API's base URI, e.g., `http://my-restapi.com`. Unfortunately, OpenAPI does not allow to attach an operation directly to the API's base URI, e.g., `GET http://my-restapi.com`, but requires at least one path segment in the endpoint's path definition. Since many web server implementations resolve a request to `GET http://my-restapi.com/` as `GET http://my-restapi.com`, the minimal path `/` is a workaround to define an endpoint for accessing the API's docroot. Therefore, '/' should not be detected as a violation of the rules specified above.
